src/components/InputField/component.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { getIn } from 'formik';
+import { Form, Input, Icon } from 'antd';
+
+const InputField = ({
+  placeholder,
+  autoComplete,
+  type,
+  icon,
+  field,
+  form: { touched, errors },
+  ...props
+}) => {
+  const fieldTouched = getIn(touched, field.name);
+  const fieldErrors = getIn(errors, field.name);
+
+  return (
+    <Form.Item
+      validateStatus={fieldErrors && fieldTouched && 'error'}
+      help={fieldErrors && fieldTouched && fieldErrors}
```

Comment:
Form.Item accepts string as text message, but you might have `false` in this construction

Reply:
In first-time fieldErrors works as condition (if it returns string than fieldErrors=true).
The second time it is message.

help={fieldErrors && fieldTouched && fieldErrors}
equals
help={fieldErrors && fieldTouched ? fieldErrors : ''}

Reply:
Fixed.